tools/GlobalToolShimGenerator/ShimGenerator.targets

```diff
@@ -0,0 +1,121 @@
+<Project DefaultTarget="GenerateShims">
+  <Import Project="$(MSBuildThisFileDirectory)\ShimGeneratorInner.targets" />
+
+  <Target Name="BuildEmbedAppNameInHostTask">
+    <!--<Message Text="*** BuildEmbedAppNameInHostTask ***" Importance="high"/>-->
+
+    <MSBuild BuildInParallel="False"
+             Targets="Restore;Build"
+             Projects="$(MSBuildThisFileDirectory)\EmbedAppNameInHostTask\EmbedAppNameInHostTask.csproj"
+             Properties="MicrosoftBuildPackageVersion=$(MicrosoftBuildPackageVersion);MicrosoftBuildUtilitiesCorePackageVersion=$(MicrosoftBuildUtilitiesCorePackageVersion)" />
+  </Target>
+
+
+  <PropertyGroup>
+      <AppTargetFramework Condition="'$(AppTargetFramework)' == '' ">netcoreapp2.1</AppTargetFramework>
+      <ShimOutputDirectoryBase Condition="'$(ShimOutputDirectoryBase)' == '' ">bin\Shims\</ShimOutputDirectoryBase>
+      <IntermediateDirectoryForShimsBase Condition="'$(IntermediateDirectoryForShimsBase)' == '' ">$(IntermediateOutputPath)</IntermediateDirectoryForShimsBase>
+      <ShimTargetName Condition="'$(ShimTargetName)' == '' ">dotnet-aspnet-codegenerator</ShimTargetName>
+      <ShimTargetFileName Condition="'$(ShimTargetFileName)' == '' ">dotnet-aspnet-codegenerator.dll</ShimTargetFileName>
+  </PropertyGroup>
+
+  <Target Name="GenerateShims" DependsOnTargets="GenerateShims1;GenerateShims2;GenerateShims3">
+    <!--<Message Text="Target = GenerateShims [master]" Importance="high"/>-->
+  </Target>
+
+  <Target Name="GenerateShims1" DependsOnTargets="BuildEmbedAppNameInHostTask">
+    <!--<Message Text="*** GenerateShims1 ***" Importance="high"/>-->
+    <PropertyGroup>
+      <Rid>win-x86</Rid>
+      <NativeExecutableExtension Condition=" $(Rid.StartsWith('win'))">.exe</NativeExecutableExtension>
```

Comment:
Always true, right?

Reply:
Yes, removing the condition from NativeExecutableExtension... leftover from a previous iteration.